packages/design-system-docs/src/pages/components/FilterTag/_FilterTag.docs.scss

```diff
@@ -0,0 +1,45 @@
+/*
+Filter Tag
+
+Filter Tags are used to display dismissible filter chips
+
+Markup: filtertag.example.html
+
+Style guide: components.filtertag
```

Comment:
We should mark this component as draft by putting `@status Draft` on the doc page